storm/ci/storm.rake

```diff
@@ -31,6 +31,8 @@ namespace :ci do
       # Deploy the basic WordCountTopology we created earlier.
       sh %(docker run --link storm-nimbus:nimbus -it --rm -v $(pwd)/topology.jar:/topology.jar storm:1.1 storm jar \
         /topology.jar org.apache.storm.starter.WordCountTopology topology)
+      # Wait again for slow topology starts.
+      sh %(sleep 60)
```

Comment:
We use use docker `healthcheck` or some command to check the state of the topology, if possible instead of a sleep. 
But lets do that in another PR to improve the CI (this one is big enough).